src/BulkAppointments/AppointmentForm.js

```diff
@@ -34,32 +34,32 @@ export const onSubmit = onSucces => values => {
   }
 
   if (!values.appointmentType) {
-    formErrors.push({ targetName: 'appointmentType', text: 'Appointment type is required' })
+    formErrors.push({ targetName: 'appointmentType', text: 'Select appointment type' })
   }
 
   if (!values.location) {
-    formErrors.push({ targetName: 'location', text: 'Location is required' })
+    formErrors.push({ targetName: 'location', text: 'Select location' })
   }
 
   if (!values.date) {
     formErrors.push({ targetName: 'date', text: 'Date is required' })
   }
 
   if (isToday && moment(values.startTime).isBefore(now)) {
-    formErrors.push({ targetName: 'startTime', text: "Start time shouldn't be in the past" })
+    formErrors.push({ targetName: 'startTime', text: 'Start time must be in the future' })
```

Comment:
this message doesn't correspond to the validation.  Start time can be now can't it?

Reply:
Good point. It was better before. I'll change it back.

Reply:
although you can't use the word `shouldn't` as it implies a choice, `must` is better